app/assets/images/payment_icons/latitude_gem_au.svg

```diff
@@ -0,0 +1 @@
S8zLXnUjpnKEeWLZDK0rOQ/eQfxHxj46kSvmmquzbMbOA6l769uvEan1/ahW5csRd36Y9SFstPucGC1zvzUILvUyLt6iSuF6gNo7krUCTc0PVCXA8yhLkBRVqJJPC7ctzlqLUgFxIiKJboipfu1q/N9Uk11/ABiIDu6gYjaE+/t6tueeQzRV6FThMX6GWEjOii6fU4mv5ZIm1QiZdmgc+vcaGcueSyzgCxwpEMIR5n358XW1IIGkgHyuiP95MjWKJ9Atr1UUPf1nLPeZDpqkI3Kxxpk+Yx5AwliaCFvRtA/v05r6+Qq1GcxgGddEDN6at+Yt+nC4Xc9DpOli92u+lm47q+wBqK3iH6/AH46QfqOf+d9Ij2xZfp3IYm6kMaqz9G7KM8+vWHyAjtgwT+0t7YEhg/2OAHSHYe83P/0iDwD1rW6p4ADusinqOSkGx/fl/kRt4m1KNWaBTb0Nt+LQEdnPf08/kz9BPve7yf5nido0nTq7di74UhlbStyJya8YUu6ENa9ufSNCF7GcYe3Mfrp0cm7OaJfmgvrt5jH8v7jxTYXz1GZkZmCl3g//2vMwaB/MPa12uM/89BYt56QEEgWrcwOE3N9/wFf5JDHfGWnYBqsNna1oX2MhSASEMi7NWf/6AP73pzCRC4AU6p3YIeTClcRyW/Za/Ginai2L/oIn2dv9uaPv5CC/LpaIoJkif/uNyfP/KHVNIxkmO+TxC2kH/al5Yi1nQ0pfw1+c8X/zOsqFNaYeXKjzaVCr8kXudgzhTugZ5ZNEEK4kVvCuLxfyy0L2GZ/IwbKxd0LRZVey8FbEPHmkOhfN0tuS4ZoSXS/sGK+c/92XdcDEELFG/9dsMFLyODShTZpUs/G21JN8ocojxiiHQMkxdERE/QPFSnGNwZ9lLGOZVVx3UPGsuFqttO/f/1OVuokkoyMnKIeyK3QU44pM+ppVhcd5w1h+OKIl1FS9ruc3yzHI52X8fukRkliS6NTG1fSSW5Uk/2OqRBQRjHXYlDudbaiLCFmqAte2uJ97j+iQY5l41VMTMiJIc6JTdRySzYl7RdyXeDyxSwM/B0U9hQ1214TdzVcsXjnoIk3EPzTUrWN5/bumjLzyeg+kqbcL1lYUVA6IbjcAYsKFx4cwHf1xwzOGd1qzj6XiqIa2kwL1HWcYbwcasmqgtpkFzoRgJweX7GFd7hbtgXp1A3tBgHyfLIMMkw5kHGhh7PS1un5nMDyzQZzVhMlhnXSYu7h+kV1CJQdw5FLDVj8fTYXJlq6O7MpBYPXtoi3SnZRDVkQ8uH5IqwMC50BMftkLxINMrPuS2SHky4mSn4SbI4UoN5vrS1yN0KD8avbVL9byMzy2hCxrzB5kzrrdV9FscIGhlZCHcPWs7sIE3+yOuUvrfuuVeR+Eu16Xvc0yp041Ic0czn4NkiY9wZzBFJo1uYTXqGWaRcx2RO5WW0590QQxzOTv2pkDQ2R3DFUhC7o887c9OwWI3KuIzIqqDASDAiMxQHp2Z+jT1ox5BBd1h5la6OcY8S5yEqhagEu6kdQJriCMt0i6hYQFpDWdgDwOuV6H4diQ7oNJSCUWZw9HenrdowiWPNbE7TyhojgzwhM0bRRBl4hwwH3GvugWvPDejm3p1JljkEbShW3PeR+ewtSYg7qZDbPVKvP7LMikAlkfkAqkXYZsyt/u3GfjUAIuFlVWZuZeXU/UpWw2uzAlQm5VBeKp/DYCzAdLsDCtlGr2UlWW8LGYi0CC4aubjcSpfTTlVgh+z/AQvzrYpBhUynAAAAAElFTkSuQmCC"/></g><path d="M0 3v18a3.009 3.009 0 003 3h32a2.946 2.946 0 003-3V3a3.009 3.009 0 00-3-3H3a2.946 2.946 0 00-3 3zm35-2a2.006 2.006 0 012 2v18a2.006 2.006 0 01-2 2H3a2.006 2.006 0 01-2-2V3a2.006 2.006 0 012-2z" fill="#002a66" fill-rule="evenodd" opacity=".3"/><path d="M12.525 12.4a.663.663 0 10.662.663.663.663 0 00-.662-.663zM6.937 8.232H5.611v7.577h7.576v-1.326h-6.25zm3.5 1.326a.663.663 0 10-.659-.658.663.663 0 00.663.658zm2.084.758a.663.663 0 10.662.663.663.663 0 00-.658-.663zm-4.163-.758A.663.663 0 107.7 8.9a.662.662 0 00.658.658zm4.167-1.326a.663.663 0 10.662.663.663.663 0 00-.662-.663zm19.914 1.925h-.422l-1.482 2.533-1.5-2.533h-.425v3.6h.49v-2.61l1.3 2.172h.236l1.3-2.188.006 2.626h.494zm-8.345 1.992H25.9v-.437h-1.806v-1.107h2.028v-.448h-2.543v3.6h2.615v-.448h-2.1zm-5.205-1.4a1.514 1.514 0 01.738-.177 1.452 1.452 0 011.046.416l.319-.319a1.6 1.6 0 00-.6-.414 2.223 2.223 0 00-1.776.1 1.748 1.748 0 00-.689.659 1.915 1.915 0 000 1.892 1.768 1.768 0 00.687.658 2.006 2.006 0 00.983.239 2.327 2.327 0 00.741-.118 1.916 1.916 0 00.623-.34v-1.4h-.494v1.174a1.524 1.524 0 01-.844.231 1.5 1.5 0 01-.734-.18 1.327 1.327 0 01-.512-.5 1.457 1.457 0 010-1.421 1.3 1.3 0 01.512-.497z" fill="#fff"/></svg>
```

Comment:
Same for this one: [![alt](https://screenshot.click/08-18-34638-mblol.png)](https://screenshot.click/08-18-34638-mblol.png)


You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.